src/app/components/organization-info/header.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import * as Constants from '../../constants/constants'
+import '../../styles/header.css';
+import { Multiselect } from 'multiselect-react-dropdown';
+
+
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedLanguages: [],
+      isVisible: false,
+    }
+    this.selectedLanguagesArr = [];
+  }
+
+  getSelectedLanguages = (event) => {
+    event.preventDefault();
+    this.props.filterRepos(this.selectedLanguagesArr);
+  }
+
+  getLanguageSelector(organization) {
+    const repositoriesLanguages = {};
+    if (organization) {
+      organization.repositories.nodes.map(repo => {
+        return (
+        repo.languages.nodes.map(lang => {
```

Comment:
Since there is no modification to each item of the array iterated and we just need to iterate the array to perform an action, so used forEach instead of map.